src/main/scala/nl/knaw/dans/easy/bagindex/package.scala

```diff
@@ -96,8 +66,7 @@ package object bagindex {
     def asScala: Option[T] = if (opt.isPresent) Some(opt.get()) else None
   }
 
-
-  // TODO: WILL BE IN dans-scala-lib
+  // TODO: will NOT be in dans-scala-lib, because it actually doesn't work correctly...
```

Comment:
So you must have goor reasons to put it here even though it doen't work correctly. Would it make sense to share your reasons with us?